{
	"id": "oval:com.redhat.rhsa:def:20170937",
	"version": "635",
	"class": "patch",
	"metadata": {
		"title": "RHSA-2017:0937: python-defusedxml and python-pysaml2 security update (Moderate)",
		"reference": [
			{
				"ref_id": "RHSA-2017:0937",
				"ref_url": "https://access.redhat.com/errata/RHSA-2017:0937",
				"source": "RHSA"
			},
			{
				"ref_id": "CVE-2016-10149",
				"ref_url": "https://access.redhat.com/security/cve/CVE-2016-10149",
				"source": "CVE"
			}
		],
		"description": "The defusedxml package contains several Python-only updates for security vulnerabilities in Python's XML libraries. Defusedxml functions and classes can be used instead of the originals to protect against entity-expansion and DTD-retrieval issues.\n\nPySAML2 is the python implementation of SAML Version 2, containing all the functionality for building a SAML2 service provider or an identity provider, to be used in a WSGI environment.\n\nSecurity Fix(es):\n\n* An XML entity expansion vulnerability was found in python-pysaml2. A remote attacker could send a crafted request which would cause denial of service through resource exhaustion. (CVE-2016-10149)",
		"advisory": {
			"from": "secalert@redhat.com",
			"severity": "Moderate",
			"updated": {
				"date": "2017-04-12"
			},
			"cve": [
				{
					"text": "CVE-2016-10149",
					"cvss3": "5.3/CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:L",
					"cwe": "CWE-776",
					"href": "https://access.redhat.com/security/cve/CVE-2016-10149",
					"impact": "moderate",
					"public": "20161031"
				}
			],
			"affected": {},
			"affected_cpe_list": {
				"cpe": [
					"cpe:/a:redhat:openstack-director:9",
					"cpe:/a:redhat:openstack-director:9::el7",
					"cpe:/a:redhat:openstack-optools:9",
					"cpe:/a:redhat:openstack-optools:9::el7",
					"cpe:/a:redhat:openstack:9",
					"cpe:/a:redhat:openstack:9::el7"
				]
			},
			"rights": "Copyright 2017 Red Hat, Inc.",
			"issued": {
				"date": "2017-04-12"
			},
			"bugzilla": [
				{
					"text": "CVE-2016-10149 python-pysaml2: Entity expansion issue",
					"href": "https://bugzilla.redhat.com/1415710",
					"id": "1415710"
				}
			]
		},
		"affected": {
			"family": "unix",
			"platform": [
				"Red Hat OpenStack Platform 9.0 (Mitaka)"
			]
		}
	},
	"criteria": {
		"operator": "OR",
		"criterias": [
			{
				"operator": "AND",
				"criterias": [
					{
						"operator": "OR",
						"criterias": [
							{
								"operator": "AND",
								"criterions": [
									{
										"test_ref": "oval:com.redhat.rhsa:tst:20170937001",
										"comment": "python-defusedxml is earlier than 0:0.5.0-1.el7ost"
									},
									{
										"test_ref": "oval:com.redhat.rhsa:tst:20170937002",
										"comment": "python-defusedxml is signed with Red Hat redhatrelease2 key"
									}
								]
							},
							{
								"operator": "AND",
								"criterions": [
									{
										"test_ref": "oval:com.redhat.rhsa:tst:20170937003",
										"comment": "python-pysaml2 is earlier than 0:3.0.2-3.el7ost"
									},
									{
										"test_ref": "oval:com.redhat.rhsa:tst:20170937004",
										"comment": "python-pysaml2 is signed with Red Hat redhatrelease2 key"
									}
								]
							}
						]
					}
				],
				"criterions": [
					{
						"test_ref": "oval:com.redhat.rhba:tst:20182140007",
						"comment": "Red Hat Enterprise Linux 7 is installed"
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:com.redhat.rhba:tst:20182140008",
				"comment": "Red Hat Enterprise Linux must be installed"
			}
		]
	}
}
